src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import classNames from 'classnames';
+import React from 'react';
+import { FilterType } from '../../types/FilterType';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filterType: FilterType,
+  setFilterType: (filter: FilterType) => void,
+  completedTodos: Todo[];
+  activeTodos: Todo[];
+  removeCompletedTodos: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  activeTodos, filterType, setFilterType, completedTodos, removeCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodos.length} items left`}
```

Comment:
Seems that you use only length value in this component. So, do you really need to pass the whole activeTodos array as a prop?